src/RequestProcessor.cs

```diff
@@ -340,6 +363,19 @@ internal StreamingMessage ProcessInvocationCancelRequest(StreamingMessage reques
         {
             return null;
         }
+        
+        private StreamingMessage ProcessFunctionMetadataRequest(StreamingMessage request)
+        {
+            // WorkerStatusResponse type says that it is not used but this will create an empty one anyway to return to the host
+            StreamingMessage response = NewStreamingMessageTemplate(
+                request.RequestId,
+                StreamingMessage.ContentOneofCase.FunctionMetadataResponse,
+                out StatusResult status);
+
+            response.FunctionMetadataResponse.FunctionMetadataResults.AddRange(WorkerIndexingHelper.IndexFunctions(request.FunctionsMetadataRequest.FunctionAppDirectory));
```

Comment:
Without knowing the specifics of the contract between the Host and worker, I can't decipher what the comment at the top of this block is trying to say. Can you please add a bit more context? What exactly "says that it is not used"?, and what is "it" in this context? Thanks!

Reply:
I have no idea what this comment is about either. My guess - some misguided note I made while working on my first prototype and forgot to remove. I have removed it